README.md

```diff
@@ -130,6 +130,93 @@ constructorio.recommendations.getUserFeaturedItems({ parameters }).then(function
 | --- | --- | --- |
 | `results` | number | Number of results to retrieve |
 
+### Tracker
+
+The tracker module can be used to send tracking events. Returns `true` when successful, or will throw an error if an issue is encountered.
+
+#### Send autocomplete select event
+```javascript
+constructorio.tracker.sendAutocompleteSelect('dogs', {
```

Comment:
We'll want to rename all of the `sendXxxEvents` to `trackXxxEvents` in the name of consistency with other code.